onnxruntime/core/session/inference_session.cc

```diff
@@ -432,10 +432,10 @@ common::Status InferenceSession::Initialize() {
 
         if (it->get()->Type() == onnxruntime::kCudaExecutionProvider) {
           LOGS(*session_logger_, ERROR) << "Parallel execution cannot be enabled "
-                                           "when CUDA execution provider is registered.";
+                                           "when CUDA execution provider is registeredas first priority.";
```

Comment:
Missing space

Reply:
I think this already got resolved in the commit - cf86dced38d7023750c173e68450bbd1c4d54ca4